src/node/tests/elder.rs

```diff
@@ -227,14 +248,16 @@ impl Env {
         }
     }
 
-    fn accumulate_section_info_if_vote(&mut self, new_info: &DkgToSectionInfo) {
-        let section_key = new_info.new_pk_set.public_key();
+    fn accumulate_our_key_and_section_info_if_vote(&mut self, new_info: &DkgToSectionInfo) {
         let _ = self.n_vote_for_gossipped(
             NOT_ACCUMULATE_ALONE_VOTE_COUNT,
-            iter::once(AccumulatingEvent::SectionInfo(
-                new_info.new_elders_info.clone(),
-                section_key,
-            )),
+            vec![
+                AccumulatingEvent::OurKey {
```

Comment:
Yes, I had to split the `SectionInfo` into two votes, because there are two distinct pieces of info that are signed - the `EldersInfo` and the section key and they need separate signatures.